code/script.js

```diff
@@ -316,10 +343,24 @@ const checkMyGuess = (personToCheck) => {
   // 2. Set a Message to show in the win or lose section accordingly
   // 3. Show the win or lose section
   // 4. Hide the game board
+
+  let messgage =""
+  if (secret.name ===personToCheck){
+    alert("Holy Moly you guessed correctly!")
+  } else {
+    alert ("Woopsy daisy better luck next time")
+  }
+  winOrLose.innerHTML=message
+  boardWrapper.classList.add("board-wrapper-inactive")
+  winOrLose.classList.add("win-or-lose-wrapper-active")
```

Comment:
One suggestion to another alternative, making the game being forced to restart upon completion(win/loss) is that you make the right html elements be hidden/shown and likeswise when afterwards, calling for a restart.
instead of using prompt, the screen could be forced to swap to a restart button only with win/loss-text above.

 winOrLose.style.display = "flex"
    board.style.display = "none"
    winOrLoseText.style.display = "block"
is one easy way to implement css commands into JS for example which can be written as is within a function.